packages/askui-nodejs/src/execution/retry-strategies/ui-controller-linear-retry-strategy.ts

```diff
@@ -1,15 +1,27 @@
 import { RetryStrategy } from '../ui-controller-retry-strategy';
 
+/**
+ * LinearRetryStrategy implements a retry strategy that uses a linear backoff algorithm.
+ */
 export class LinearRetryStrategy implements RetryStrategy {
-  baseDelayMs: number;
+  baseDelayMs: number; // Base delay in milliseconds for each retry
 
-  retryCount: number;
+  retryCount: number; // Maximum number of retry attempts
```

Comment:
I would remove the inline comments. They don't provide any more valuable information than the jsdocs.